webgrid/testing.py

```diff
@@ -277,12 +277,29 @@ class GridBase:
     grid_cls = None
     filters = ()
     sort_tests = ()
+    _url = '/'
 
     @classmethod
     def setup_class(cls):
         if hasattr(cls, 'init'):
             cls.init()
 
+    @property
+    def url(self):
+        return self._url
+
+    @contextmanager
+    def url_context(self, url):
```

Comment:
I'm open to other approaches if this context manager seems a bit weird. It seemed like the best way to ensure that all calls to get_session_grid used the same URL for the request context, without explicitly passing the url to each dependent method.

Reply:
I'd rather not - we're trying to shortcut the need to use a context manager to call `get_session_grid`. Instead, let's have a special kwarg to pass to `get_session_grid`, like `_query_string`, and just pass that into the request context.